components/metatext_ai_inference_api/metatext_ai_inference_api.app.mjs

```diff
@@ -0,0 +1,53 @@
+import { axios } from "@pipedream/platform";
+import constants from "./common/constants.mjs";
+
+export default {
+  type: "app",
+  app: "metatext_ai_inference_api",
+  propDefinitions: {
+    modelType: {
+      type: "string",
+      label: "Model Type",
+      description: "The type of model to use for the request.",
```

Comment:
```suggestion
      description: "The type of model to use for the request.",
```

Reply:
Changed "the" to "The" in the description to follow Google's Developer docs style guide (Capitalize most words in titles and headings)